### PR TITLE
LPS-142078 Compute new labels for stylebook inheritance

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:test:echo

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/select_style_book.jsp [hunk 1]
```diff
@@ -68,17 +68,17 @@ List<StyleBookEntry> styleBookEntries = selectLayoutPageTemplateEntryDisplayCont
 	var delegateHandler = delegate(
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/select_style_book.jsp:57]: Maybe we are ready to replace this with const. And maybe we can move it out of the JSP file.

(no tiene nada que ver con tu PR? correcto)
(te ha tocado? parece que sí)
(seguimos el reglamento de los boy scouts)

**Comment by p2kmgcl** [comment]
Looks really nice. I just suggested a possible refactor that we can apply to select_style_book.jsp, but the PR looks perfect :partying_face:

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -14143,6 +14143,9 @@ style-book-name=Style Book Name
 style-book-thumbnail=Style Book Thumbnail
 style-guide-sample=Style Guide Sample
 styles=Styles
+styles-by-default=Styles By Default
```
Comment (ealonso) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:14146]: By should be by.

cc: @victorg1991

File: modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/display/context/StyleBookDisplayContext.java [hunk 3]
```diff
@@ -109,14 +112,32 @@ public PortletURL getPortletURL() {
 					themeDisplay.getScopeGroupId(), _getKeywords()));
 		}
 		else {
-			styleBookEntriesSearchContainer.setResultsAndTotal(
-				() -> StyleBookEntryLocalServiceUtil.getStyleBookEntries(
-					themeDisplay.getScopeGroupId(),
-					styleBookEntriesSearchContainer.getStart(),
-					styleBookEntriesSearchContainer.getEnd(),
-					styleBookEntriesSearchContainer.getOrderByComparator()),
+			List<StyleBookEntry> styleBookEntries = new ArrayList<>();
```
Comment (ealonso) [inline modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/display/context/StyleBookDisplayContext.java:115]: Could you please make it consistent with MasterLayoutDisplayContext? @victorg1991

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties (hunk 2 above)
Comment (victorg1991) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:14146]: Thanks!

File: modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/display/context/StyleBookDisplayContext.java (hunk 3 above)
Comment (victorg1991) [reply modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/display/context/StyleBookDisplayContext.java:115]: 🤔 I'll review that

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/select_style_book.jsp (hunk 1 above)
Comment (victorg1991) [reply modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/select_style_book.jsp:57]: I wil :)

Comment (victorg1991) [reply modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/select_style_book.jsp]: I've done it :D

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties (hunk 2 above)
Comment (victorg1991) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:14146]: Done!

File: modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/display/context/StyleBookDisplayContext.java (hunk 3 above)
Comment (victorg1991) [reply modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/display/context/StyleBookDisplayContext.java:115]: Write you over slack, it seems that the masterLayoutDisplayContext is not working properly

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:test:echo

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:test:echo

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:test:echo

**Comment by ealonso** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by victorg1991** [comment]
resent here: https://github.com/liferay-echo/liferay-portal/pull/7140

**Pull request closed without merging**